rbac/ledger_sync/inbound/listener.py

```diff
@@ -46,7 +46,11 @@ def process(rec, conn):
         batch = batch_pb2.Batch()
         batch.ParseFromString(rec["batch"])
         batch_list = batch_to_list(batch=batch)
-        status = ClientSync().send_batches_get_status(batch_list=batch_list)
+        client = ClientSync()
+        status = client.send_batches_get_status(batch_list=batch_list)
+        while status == "PENDING":
```

Comment:
Look at line 54, the variable `status` will not contain the value `PENDING`. You would probably have to copy line 54 and have it
`while status[0]["status"] == "PENDING"`